docs/New Game.md

```diff
@@ -147,6 +147,35 @@ Define a [JSON Schema](https://json-schema.org/understanding-json-schema/) that
 This makes for a good documentation of what is the input for the patcher, as well as a very good safety against
 misconfiguration in Randovania.
 
+# Expectations of a Development Game
+
+After a game has been merged and is available in development builds, the responsibility of maintaining it is now shared between you and the Randovania team:
+- Randovania: We make sure your game is included in all Randovania changes.
+- You: Work towards the game reaching stable.
+
+We're happy to support new games by doing our part, but since that has a cost, that we don't want to be doing forever, there are some expectations regarding time frames:
+
+### Active Development
+
+Please don't go for too long without visible development of either patcher or non-logic Randovania changes.
+
+### Exportable from Dev Builds
+
+After roughly one year, we expect a game to be exportable directly from a regular dev build.
+How good the experience is when playing a seed is up to your discretion.
+We do however recommend integrating it as early as it is possible, for a better developer experience.
+
+### Becoming Stable
+
+After roughly two years, we expect a game to be close to being considered stable.
+
+## Remarks
+
+All of these only apply to games in development. After a game reaches stable, further development is not needed.
+
+Please keep us in the loop of what's going on - if you'll need to step away for some time, if you're having trouble with anything and so on.
```

Comment:
This is probably a good spot to ammend the "potential to get removed" part.

```suggestion
Please keep us in the loop of what's going on - if you'll need to step away for some time, if you're having trouble with anything and so on.
We may remove games from the codebase after they have been merged, if we feel that there no work done from your side towards making the game stable. This does not mean that we reject the game! It can always come back, but we don't want to maintain something that does not feel worth being maintained.
```

Reply:
Don't like "worth being maintained". let me think